src/requests/getForecast.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+import axios from "axios";
+
+const getForecast = async (
+  searchText,
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  setErrorMessage
+) => {
+  let endpoint = "https://cmd-shift-weather-app.onrender.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+
+  await axios
+    .get(endpoint)
+    .then((res) => {
+      setErrorMessage("");
+      setSelectedDate(res.data.forecasts[0].date);
+      setForecasts(res.data.forecasts);
+      setLocation(res.data.location);
+    })
+    .catch((error) => {
+      const { status } = error.response;
+      if (status === 404) {
+        console.error("Invalid location", error);
```

Comment:
Since you are already displaying an error message to the user explaining why things are not working, these console logs could be removed
You could also remove the eslint no console exception at the top of the file as a result of this too